spec/models/comment_spec.rb

```diff
@@ -37,4 +37,23 @@
     comment = build(:comment, markeddown_comment: "a" * 16_777_216)
     expect(comment).to_not be_valid
   end
+
+  describe ".accessible_to_user" do
+
+    context "when user is a moderator" do
+      let(:moderator) { create :user, :moderator }
```

Comment:
Please rewrite these to just be `it` tests, move the creation and such inside. I try to only use `let` in the codebase when there's a very generic object that many tests want to use without caring very much about.